src/utils/solve-grid/index.ts

```diff
@@ -1,35 +1,50 @@
-import global from 'global'
-import { GRID, NUMBERS } from 'typings'
-import { checkGrid, identifySquare, isInCol, isInRow, isInSquare } from 'utils'
+import global from "../../global";
+import { GRID, NUMBERS } from "../../typings";
+import {
+  checkGrid,
+  identifySquare,
+  isInCol,
+  isInRow,
+  isInSquare,
+} from "../../utils";
 
-const numbers: NUMBERS[] = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+const numbers: NUMBERS[] = [1, 2, 3, 4, 5, 6, 7, 8, 9];
 
 function solveGrid(grid: GRID) {
-  let row = 0
-  let col = 0
+  let row = 0;
+  let col = 0;
 
   for (let i = 0; i < 81; i++) {
-    row = Math.floor(i / 9)
-    col = i % 9
-
+    row = Math.floor(i / 9);
+    col = i % 9;
+    /* eslint-disable */
+    /**
+     * eslint prefer-const not let
+     */

```

Comment:
Same comment as above regarding disabling eslint rules. That said, did you try changing the let for a const and seeing if that fixed the rule and kept the code working?

Reply:
ja tried, but not correct, cuz I want to change the initial value latter, not for all